DashSync/shared/Models/Chain/DSChain.m

```diff
@@ -405,6 +405,14 @@ - (void)setUp {
 
 // MARK: - Helpers
 
+- (BOOL)isCore19Active {
+    return self.lastTerminalBlockHeight >= chain_core19_activation_height(self.chainType);
+}
+
+- (KeyKind)getActiveBLSType {
```

Comment:
@pankcuf Avoid using "get" in naming (https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html)

Reply:
ok, will rename it after